packages/account/src/Configs/financial-details-config.ts

```diff
@@ -1,7 +1,13 @@
 import { localize } from '@deriv/translations';
 import { generateValidationFunction, getDefaultFields } from '@deriv/shared';
+import { TSchema } from 'Types';
 
-const financial_details_config = ({ financial_assessment }) => {
+type TFinancialDetailsConfig = {
+    real_account_signup_target: string;
+    financial_assessment: object;
```

Comment:
```suggestion
    financial_assessment: GetFinancialAssessment;
```
this will be needed:
`import type { GetFinancialAssessment } from '@deriv/api-types';`